mmedit/datasets/sr_vimeo90k_multiple_gt_dataset.py

```diff
@@ -57,10 +56,14 @@ def load_annotations(self):
 
         data_infos = []
         for key in keys:
-            lq_paths = sorted(
-                glob.glob(osp.join(self.lq_folder, key, '*.png')))
-            gt_paths = sorted(
-                glob.glob(osp.join(self.gt_folder, key, '*.png')))
+            lq_paths = [
+                osp.join(self.lq_folder, key, f'im{i}.png')
```

Comment:
Unrelated to this pr, but the hardcoded png format is a headache also.

Reply:
It is the official format of Vimeo-90K, I think it is reasonable to take this format (?)

Reply:
Not really. Our codebase is for developers, who will use their own format (whatever their dataset happens uses)

Reply:
This class is for Vimeo-90K only. `SRFolderVideoDataset` or `SRFolderMultipleGTDataset` should be used for general purposes.

Actually we could support `start_idx` and `filename_tmpl`. But this way cannot support general cases. For instance, it fails when the files are named `im01.png, im03.png, im05.png, im07.png, im09.png, img11.png, img13.png`.

Reply:
Okay lets keep it as is for now.